samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/tests/test_cookiecutter.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Similar to the other comment, you can remove this file as we're not running integ tests and they're now being cleaned up in another PR :)